raven/base.py

```diff
@@ -577,10 +580,10 @@ def _log_failed_submission(self, data):
         if 'exception' in data and 'stacktrace' in data['exception']['values'][0]:
             # try to reconstruct a reasonable version of the exception
             for frame in data['exception']['values'][0]['stacktrace']['frames']:
-                output.append('  File "%(filename)s", line %(lineno)s, in %(function)s' % {
-                    'filename': frame['filename'],
+                output.append('  File "%(fn)s", line %(lineno)s, in %(func)s' % {
```

Comment:
Why'd you change this? I know the end result is the same, just curious since it's unrelated.